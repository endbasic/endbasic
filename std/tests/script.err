ERROR: 22:1: Unknown builtin HELP
