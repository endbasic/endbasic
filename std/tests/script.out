The random number is: 70
